content/dev_stories/playground_to_production_fruit_falling.md

```diff
@@ -0,0 +1,677 @@
+whole development process.**
+1.	Once I had something I was ready to share, I wanted to host it using 
+[GitHub Pages](https://pages.github.com/) so I could get feedback from 
+friends and colleagues. Because the Babylon.js Template Repositories are 
+set up to make this easy, **it took me five minutes, if that, to publish my 
+experience to a GitHub-hosted site I could link to.**
+1.	Next, I uploaded my game code as an 
+[NPM package](https://www.npmjs.com/) so that I could easily re-add it to 
+other projects (websites, NPM apps, etc.) just by pulling down a package. 
+Again, the Babylon.js Template repositories are set up to make this easy: 
+**within ten minutes, I had my new NPM package.**
+1.	I wanted to publish my app as a mobile game and get it listed on Google 
+Play and the App Store, so I decided to use 
+[Ionic](https://ionicframework.com/). All I had to do was create a new 
+Ionic app, add my new NPM package as a dependency, then invoke my code and 
+tell it where to display in the app. **About an hour later, my code was 
+built into a native app, ready to be shipped for mobile devices or as a 
+PWA.**
+
+![The Path: Playground to GitHub to NPM to Ionic](/img/devVignettes/fruitFalling/answer.jpg!800)
+
+And that is how I created 
+[_Fruit Fallin'_](https://syntheticmagus.github.io/fruit-falling-source/), 
+a Babylon.js app inspired by browser games from the heyday of Flash. While 
+I happened to be making a simple game, this same process can be used for 
+more complex experiences, commercial scenarios... At a high level, the 
+procedure described above can be used to take any Babylon.js idea from 
+Playground to production using the Babylon.js Template Repositories, NPM, 
+and Ionic.
+
+For a less high-level look, let's take a more step-by-step journey through 
+the process.
+
+## A More Step-by-Step Journey Through the Process
+
+### Moving Playground Code into a Development Repo
+
+The first step toward developing production code was to move development 
+into a dedicated repository. The Babylon.js NPM Package Template is 
+designed with this in mind, particularly when migrating code from the 
+Babylon.js Typescript Playground, so I decided to use that.
+
+1.  I logged into my GitHub account.
+
+    ![GitHub login](/img/devVignettes/fruitFalling/00_log_in_to_github.jpg!250)
+1.  On the page for the Babylon.js 
+    [NPM Package Template](https://github.com/BabylonJS/npm-package-template), 
+    I clicked the "Use this template" button to start a new repo.
+
+    !["Use this template" button](/img/devVignettes/fruitFalling/01_use_package_template.jpg!500)
+1.  I picked my repo name, choose to make it public (for non-open source, 
+    I would have chosen private), then created the repository.
+
+    ![Picking repo options](/img/devVignettes/fruitFalling/02_choose_options.jpg!500)
+1.  In a Git-enabled terminal (in my case, Git Bash on Windows), I 
+    navigated to where I wanted my new repo to be and created my local 
+    clone.

```

Comment:
Maybe linking this doc to help users with less git experience? https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository